ams/BaseMigrator.cs

```diff
@@ -164,5 +178,10 @@ await foreach (var value in reader.ReadAllAsync(cancellationToken))
 
             return resourceFilter;
         }
+     private Task<MediaServicesAccountResource> GetMediaAccountAsync(string mediaAccountName, CancellationToken cancellationToken)
+        {
+            return _resourceProvider.GetMediaAccountAsync(mediaAccountName, cancellationToken); ;
+        }
```

Comment:
There is no need to add a new private function which contains a single line here.

You can just move below line  to line 68, and remove this function.

_resourceProvider.GetMediaAccountAsync(mediaAccountName, cancellationToken);